src/photos/components/PhotoList.jsx

```diff
@@ -64,10 +69,8 @@ export class PhotoList extends Component {
       }
     )
 
-    const photoIds = photos.map(p => p.id)
-    const allSelected =
-      selected.length === photoIds.length &&
-      selected.every(id => photoIds.indexOf(id) !== -1)
+    const selectedIds = selected.map(p => p._id)
+    const allSelected = photos.every(p => selectedIds.indexOf(p._id) !== -1)
```

Comment:
what is the difference betweet `selected` and `allSelected` ?

Reply:
`allSelected` = `areAllPhotosSelected`

Reply:
ok, so best to rename it so it is more explicit

Reply:
du coup ça mérite de renommer je pense, que ce soit explicite

Reply:
Shal we rename this boolean variable into something more boolean-explicit: `isEverythingSelected` for instance